build_tools/cmake/iree_hal_cts_test_suite.cmake

```diff
@@ -23,6 +23,8 @@ include(CMakeParseArguments)
 #       `-iree-hal-target-backends` option of `iree-compile` to use for
 #       executable generation. If this is omitted, or the associated compiler
 #       target is not enabled, tests which use executables will be disabled.
+#   COMPILER_ARGS: Additional compiler arguments.
```

Comment:
COMPILER_FLAGS for consistency with other rules

Reply:
I renamed it.